recipes/mender-setup/debian-master/rules

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@
+
+override_dh_auto_install:
+	DESTDIR=debian/mender-setup make install
+
+override_dh_auto_test:
+	true
+
+override_dh_strip:
```

Comment:
Q: Why do we override the `strip` section?

Reply:
To be honest I don't know. We do it for all the other golang packages ([example](https://github.com/mendersoftware/mender-dist-packages/blob/88692eea5026d274162a5828f91a076c7ebba449/recipes/mender-connect/debian-master/rules#L17)).